src/storage/v2/delta.hpp

```diff
@@ -267,6 +276,7 @@ struct Delta {
       Action action;
       PropertyId key;
       storage::pmr::PropertyValue *value = nullptr;
+      Vertex *out_vertex{nullptr};  // Used by edge's delta to easily rebuild the edge
```

Comment:
Check if this makes Deltas larger. If it does then I'd rather make a new structure 
```cpp
struct EdgeProperty {
   storage::pmr::PropertyValue pv;
   Vertex *out_vertex{nullptr};  
};
```
New Action::SET_EDGE_PROPERTY, and new field edge_property.

Reply:
As far as I know, it does not make it larger. `vertex_edge` is also 24 bytes.